mango-python/requirements.txt

```diff
@@ -3,3 +3,5 @@ matplotlib == 2.0.2
 ipython == 5.5.0
 jupyter == 1.0.0
 bdgenomics.adam == 0.24.0
+pip install nbsphinx==0.3.3
```

Comment:
Shouldn't these be in the docs/requirements.txt file? Does it have to be here as well?

Reply:
also, remove 'pip install'